launchable/commands/helper.py

```diff
@@ -16,6 +16,7 @@ def find_or_create_session(
     is_observation: bool = False,
     links: List[str] = [],
     is_no_build: bool = False,
+    lineage: Optional[str] = None,
 ) -> Optional[str]:
     """Determine the test session ID to be used.
 
```

Comment:
[nits]
I think it is a good chance that we can add missing args in this comment, from flavor to lineage.